<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMBEDevicesDialog</class>
 <widget class="QDialog" name="AMBEDevicesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMBE devices control</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QListWidget" name="ambeSerialDevices">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>List of available AMBE serial devices available</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ambeDeviceRefs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>List of devices/servers in use</string>
   </property>
  </widget>
  <widget class="QLabel" name="ambeSerialLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial devices</string>
   </property>
  </widget>
  <widget class="QLabel" name="ambeServerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>AMBE server IP and port or direct input</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ambeAddressText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>AMBE server address as ip:port or direct input</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshAmbeList">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>28</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Refresh list of devices and servers in use</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/recycle.png</normaloff>:/recycle.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="removeAmbeDevice">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>24</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Release all devices and servers</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/bin.png</normaloff>:/bin.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="ambeSerialLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>In use</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importSerial">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>24</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use serial device</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/arrow_up.png</normaloff>:/arrow_up.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="importAddress">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>24</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use server</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/arrow_down.png</normaloff>:/arrow_down.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="clearAmbeList">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>28</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Remove all devices</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/sweep.png</normaloff>:/sweep.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="statusText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>291</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importAllSerial">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>24</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use all serial devices</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/double_arrow_up.png</normaloff>:/double_arrow_up.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AMBEDevicesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AMBEDevicesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
